library/src/morpheus-master/sass/base/_defaults.scss

```diff
@@ -143,6 +143,7 @@ input[type="checkbox"], input[type="radio"]{
 	border: 1px solid var(--sakai-text-color-1);
 	color: var(--sakai-text-color-1);
 	display: inline-block;
+	margin-right: 10px;
```

Comment:
This change affects too many elements throughout Sakai. 

How about a more-specific change in _forums.scss, such as adding:
```
.Mrphs-sakai-forums #revise input[type="radio"] {
 margin: 0 4px;
}
```
And changing:
```
.Mrphs-sakai-forums #revise .checkbox input[type="checkbox"], .Mrphs-sakai-forums #revise .checkbox .Mrphs-sakai-samigo .dataTables_wrapper .dataTables_paginate > a.paginate_button[type="checkbox"], .Mrphs-sakai-samigo .dataTables_wrapper .Mrphs-sakai-forums #revise .checkbox .dataTables_paginate > a.paginate_button[type="checkbox"], .Mrphs-sakai-forums #revise .checkbox .Mrphs-sakai-samigo #authorIndexForm\:remove-selected.disabled[type="checkbox"], .Mrphs-sakai-samigo .Mrphs-sakai-forums #revise .checkbox #authorIndexForm\:remove-selected.disabled[type="checkbox"] {
- margin: 0 10px;
+ margin: 0 4px;
}
.Mrphs-sakai-forums #revise .checkbox input[type="checkbox"] + label, .Mrphs-sakai-forums #revise .checkbox .Mrphs-sakai-samigo .dataTables_wrapper .dataTables_paginate > a.paginate_button[type="checkbox"] + label, .Mrphs-sakai-samigo .dataTables_wrapper .Mrphs-sakai-forums #revise .checkbox .dataTables_paginate > a.paginate_button[type="checkbox"] + label, .Mrphs-sakai-forums #revise .checkbox .Mrphs-sakai-samigo #authorIndexForm\:remove-selected.disabled[type="checkbox"] + label, .Mrphs-sakai-samigo .Mrphs-sakai-forums #revise .checkbox #authorIndexForm\:remove-selected.disabled[type="checkbox"] + label {
- padding-left: 10px;
+ padding-left: 4px;
}
```